### DISCUSSION TITLE
Disable authentication / Cookie authentication

### DISCUSSION THREAD

**Discussion by ytzelf** [photoview/photoview#454] [Ideas]
I'm behind an auth portal and having to login twice would be annoying - any chance to have the option to disable the authentication altogether ? If the app is truly multitenant (e.g. albums & pics are attached to a user and not an instance) then maybe fetch user id (email) through http headers (see what's healthchecks doing at the REMOTE_USER_HEADER entry at https://healthchecks.io/docs/self_hosted_configuration/) ?

Thanks a lot

**Comment by viktorstrate** [comment]
Maybe #272 could solve this?
